docs/2023/laas/updates/2023-06-02.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Coding Period 
+author: Kavya Shukla
+author url: https://github.com/k-avy
+tags: [license as a service, open source licensing]
+---
+
```

Comment:
You have to add a licensing text to all the markdown files as well. Please see other's PR.